ClinGen/clincoded
# Connects to #1474. Connects to #1517. Connects to #1520. Connects to #1521. Connects to #1522. Connects to #1559. Added copy button functionality to Demographics and Methods sections.

Steps to test #1474 & #1517:
1. Select a gene-disease record in the GCI.
2. Click the "Individual" header to add individual evidence.
3. Select the "Sex" drop-down to confirm that the "Unknown" option is third and that there's a blue alert note below the field.

Steps to test #1520 & #1522:
1. Select a gene-disease record in the GCI.
2. Add a new group with some demographics and methods data.
3. With the group created, add a new family or individual using the "Add new ... to this Group" links.
4. Scroll to the Demographics and Methods sections to confirm that a copy button is available and that clicking on it updates the appropriate fields.

Steps to test #1521:
1. Select a gene-disease record in the GCI.
2. Click one of the headers (like "Family") to add evidence.
3. Scroll to the Methods section to confirm that the three specified fields have been removed and that the "Description of genotyping method" field has the specified placeholder text.
